rust/windows-client/src-tauri/src/client/logging.rs

```diff
@@ -67,8 +72,20 @@ pub(crate) async fn export_logs(managed: tauri::State<'_, Managed>) -> StdResult
         .map_err(|e| e.to_string())
 }
 
-pub(crate) async fn clear_logs_inner() -> Result<()> {
-    todo!()
+/// Delete all files in the logs directory.
+///
+/// This includes the current log file, so we won't write any more logs to disk
+/// until the file rolls over or the app restarts.
```

Comment:
Hmm interesting side effect. I suspect we'll want to handle this edge case properly if it's not a ton more work. Would it be possible to just call `callbacks.roll_log_file` here similar to the Linux client?

https://github.com/firezone/firezone/blob/main/rust/linux-client/src/main.rs#L40

Reply:
I added this as a task on #3211, it's hard to get the logging handle in the right place until that's closed.

Reply:
sgtm